Modules/Core/ImageFunction/include/itkWindowedSincInterpolateImageFunction.h

```diff
@@ -343,14 +343,14 @@ class ITK_TEMPLATE_EXPORT WindowedSincInterpolateImageFunction : public Interpol
   TWindowFunction m_WindowFunction;
 
   /** Size of the offset table */
-  static constexpr unsigned int m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
+  static constexpr SizeValueType m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
 
   /** The offset array, used to keep a list of relevant
    * offsets in the neihborhoodIterator */
-  unsigned int m_OffsetTable[m_OffsetTableSize];
+  SizeValueType m_OffsetTable[m_OffsetTableSize];
 
   /** Index into the weights array for each offset */
-  unsigned int m_WeightOffsetTable[m_OffsetTableSize][ImageDimension];
+  SizeValueType m_WeightOffsetTable[m_OffsetTableSize][ImageDimension];
```

Comment:
Also here I'm wondering, is this particular change really needed for VS2017? It will typically double the size of the table.

Reply:
I accepted your suggestion of initialization via assignment operator.